bin/helpers/utils.js

```diff
@@ -1286,6 +1290,32 @@ exports.setConfig = (bsConfig, args) => {
   }
 }
 
+// set configs if enforce_settings is passed
+exports.setEnforceSettingsConfig = (bsConfig) => {
+  let config_args = (!this.isUndefined(bsConfig) && !this.isUndefined(bsConfig.run_settings) && !this.isUndefined(bsConfig.run_settings.config)) ? bsConfig.run_settings.config : undefined;
+  if ( this.isUndefined(config_args) || !config_args.includes("video") ) {
```

Comment:
Condition for `bsConfig.run_settings` might be required here I believe?

Reply:
This method is called only upon run_settings.enforce_settings is true. Hope that check wont be needed.